mail/chrome/messenger/aboutSupportMail.dtd

```diff
@@ -15,5 +15,6 @@
 <!ENTITY aboutSupport.accountsConnSecurity            "Bezpieczeństwo połączenia">
 <!ENTITY aboutSupport.accountsAuthMethod              "Metoda uwierzytelniania">
 <!ENTITY aboutSupport.accountsDefault                 "Domyślne?">
+<!ENTITY aboutSupport.identityName                    "Tożsamość">
```

Comment:
So I will do that in a new PR after we push this one. I guess we don't want to clutter this one with whitespace noise.